sunscreen_compiler_common/src/transforms/common_subexpression_elimination.rs

```diff
@@ -174,167 +174,404 @@ mod tests {
         fn is_ordered(&self) -> bool {
             false
         }
+
+        fn is_multiplication(&self) -> bool {
+            matches!(self, Operation::Mul)
+        }
     }
 
     fn get_graph() -> CompilationResult<Operation> {
+        #[cfg(not(feature = "debugger"))]
         fn make_node(operation: Operation) -> NodeInfo<Operation> {
             NodeInfo { operation }
         }
 
+        #[cfg(feature = "debugger")]
+        fn make_node(operation: Operation, group_id: u64, stack_id: u64) -> NodeInfo<Operation> {
+            NodeInfo {
+                operation,
+                group_id,
```

Comment:
You can just hard code group_id and stack_id to 0 here and keep the same interface regardless of the debugger feature. Then you can just keep the existing code the same.